akka-stream-tests/src/test/scala/akka/stream/scaladsl/SourceSpec.scala

```diff
@@ -26,16 +26,25 @@ class SourceSpec extends StreamSpec with DefaultTimeout {
   implicit val materializer = ActorMaterializer()
   implicit val config = PatienceConfig(timeout = Span(timeout.duration.toMillis, Millis))
 
-  "Single Source" must {
-    "produce element" in {
+   "Single Source" must {
+
+     "produce exactly one element" in  {
+       //#source-single
+       val s: Future[immutable.Seq[Int]] = Source.single(1).runWith(Sink.seq)
+       s.futureValue should ===(immutable.Seq(1))
+
+       //#source-single
+     }
+
+     "produce element" in {
       val p = Source.single(1).runWith(Sink.asPublisher(false))
       val c = TestSubscriber.manualProbe[Int]()
       p.subscribe(c)
       val sub = c.expectSubscription()
       sub.request(1)
       c.expectNext(1)
       c.expectComplete()
-    }
+     }
```

Comment:
I think this introduces a formatting inconsistency, which Jenkins will complain about. You can fix it with 'sbt scalariformFormat test:scalariformFormat'